sign_oca/models/sign_oca_template.py

```diff
@@ -24,6 +24,7 @@ class SignOcaTemplate(models.Model):
     model = fields.Char(compute="_compute_model", compute_sudo=True, store=True)
     active = fields.Boolean(default=True)
     request_ids = fields.One2many("sign.oca.request", inverse_name="template_id")
+    create_single_request = fields.Boolean(default=True)
```

Comment:
I don't understand the need for this field; I think it just complicates the (clear) logic that currently exists.